CodeEdit/TabBar/TabBarItem.swift

```diff
@@ -254,31 +258,29 @@ struct TabBarItem: View {
                     )
                     .animation(.easeInOut(duration: 0.08), value: isHovering)
             } else {
-                if isActive {
-                    EffectView(
-                        // Use `windowBackground` material to fit the material of transparent titlebar
-                        // in order to make it native-styled.
-                        NSVisualEffectView.Material.windowBackground,
-                        blendingMode: NSVisualEffectView.BlendingMode.withinWindow
-                    )
-                } else {
-                    EffectView(
-                        NSVisualEffectView.Material.titlebar,
-                        blendingMode: NSVisualEffectView.BlendingMode.withinWindow
-                    )
-                    .overlay(
-                        Color(nsColor: .black)
-                            .opacity(colorScheme == .dark ? 0.50 : 0.05)
-                            .padding(.top, 1)
-                            .padding(.horizontal, 1)
-                    )
-                    .overlay(
-                        Color(nsColor: colorScheme == .dark ? .white : .black)
-                            .opacity(isHovering ? 0.05 : 0.0)
-                            .animation(.easeInOut(duration: 0.10), value: isHovering)
-                            .padding(.top, 1)
-                            .padding(.horizontal, 1)
-                    )
+                EffectView(
+                    material: NSVisualEffectView.Material.titlebar,
+                    blendingMode: NSVisualEffectView.BlendingMode.withinWindow
+                )
```

Comment:
Please pull the latest `main` since this has changed to:

```swift
EffectView(
    NSVisualEffectView.Material.titlebar,
    blendingMode: NSVisualEffectView.BlendingMode.withinWindow
)
```

and that is the reason the automated tests failed.